Makefile

```diff
@@ -71,7 +71,7 @@ dev-depext:
 	opam depext -y $(TEST_DEPS)
 
 melange:
-	opam pin add melange https://github.com/melange-re/melange.git#246e6df78fe3b6cc124cb48e5a37fdffd99379ed
+	opam pin add melange https://github.com/melange-re/melange.git#b481d7bc233bdbe453becdea6d82df64f5389d35
```

Comment:
let's update to [4025c75e765db00d9d353bf67530132709b7901c](https://github.com/melange-re/melange/commit/c04d0441b3aa0ba8a1d5cd10f6e92d380286b8ae) at least to get the fix in https://github.com/melange-re/melange/pull/398